code/datums/diseases/chronic_illness.dm

```diff
@@ -0,0 +1,103 @@
+/datum/disease/chronic_illness
+	name = "Hereditary Manifold Sickness"
+	max_stages = 5
+	spread_text = "Unspread Illness"
+	spread_flags = DISEASE_SPREAD_NON_CONTAGIOUS
+	disease_flags = CHRONIC
+	process_dead = TRUE
+	stage_prob = 0.25
+	cure_text = "Sansufentanyl"
+	cures = list(/datum/reagent/medicine/sansufentanyl)
+	infectivity = 0
+	agent = "Quantum Entanglement"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	desc = "A disease discovered in an Interdyne laboratory caused by the subjection to timesteam correction technology."
+	severity = DISEASE_SEVERITY_HARMFUL
+
+/datum/disease/chronic_illness/stage_act(seconds_per_tick, times_fired)
+	. = ..()
+	if(!.)
+		return
+
+	switch(stage)
+		if(2)
+			if(SPT_PROB(0.5, seconds_per_tick))
+				to_chat(affected_mob, span_warning("You feel dizzy."))
+				affected_mob.adjust_confusion(6 SECONDS)
+			if(SPT_PROB(0.5, seconds_per_tick))
+				to_chat(affected_mob, span_notice("You look at your hand. Your vision blurs."))
+				affected_mob.set_eye_blur_if_lower(10 SECONDS)
+		if(3)
+			if(SPT_PROB(0.5, seconds_per_tick))
+				to_chat(affected_mob, span_danger("You feel a very sharp pain in your chest!"))
+				if(prob(45))
+					affected_mob.vomit(20)
```

Comment:
friendly reminder that vomiting is a lengthy hardstun

this probably shouldn't be on stage 3